iphone/Maps/UI/BottomMenu/TabBar/BottomTabBarButton.swift

```diff
@@ -0,0 +1,33 @@
+import UIKit
+
+final class BottomTabBarButton: MWMButton {
+  @objc override func applyTheme() {
+    styleName = "BottomTabBarButton"
+    
+    for style in StyleManager.shared.getStyle(styleName)
+      where !style.isEmpty && !style.hasExclusion(view: self) {
+      BottomTabBarButtonRenderer.render(self, style: style)
+    }
+  }
+}
+
+class BottomTabBarButtonRenderer {
+  class func render(_ control: BottomTabBarButton, style: Style) {
+    UIViewRenderer.renderShadow(control, style: style)
+    UIViewRenderer.renderBorder(control, style: style)
+    
+    if let coloring = style.coloring {
```

Comment:
What is the benefit of using `if let` here and below instead of `if`?

Reply:
It is optional unwrapping  :) Because `style.coloring` returns the optional value (I am trying to use the same approach as in other renderers)